assets/websocket.js

```diff
@@ -0,0 +1,16 @@
+window.addEventListener("load", function(evt) {
```

Comment:
Currently this is called on every page; the execution should be restricted to the results page if results are still unavailable.

Reply:
Opened issue #97